### PR TITLE
move "Security" to "/learn/security"

### PR DESCRIPTION
**Pull request by sylwiavargas**
This seems a better home for this page as it does pertain to how Inngest works.

Screenshot 2024-05-24 at 2 01 19 PM

### CODE REVIEW

**Comment by sylwiavargas** [comment]
@djfarrelly how do you feel about this change?

File: next.config.mjs [hunk 1]
```diff
@@ -36,6 +36,9 @@ const permanentRedirects = [
   ["/uses/zero-infra-llm-ai", "/ai"],
   ["/uses/internal-tools", "/uses/workflow-engine"],
   ["/uses/user-journey-automation", "/blog/lifecycle-emails-with-resend"],
+
+  // new IA
+  ["/docs/security", "docs/learn/security"],
```
Comment (djfarrelly) [inline next.config.mjs:41]:
```suggestion
  ["/docs/security", "/docs/learn/security"],
```

**Review by djfarrelly — APPROVED** [review]
I think this works - feels like it's related to deeper system investigation rather than learning the basics, so happy for this to be relocated. 

Left one required change prior to merging to fix the redirect

File: next.config.mjs (hunk 1 above)
Comment (sylwiavargas) [reply next.config.mjs:41]: Ohhhh good catch, thank you!

**Pull request merged**